pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1584,6 +1584,9 @@ func getDetachedDisks(oldDisks, newDisks []api.Disk) []api.Disk {
 		if !isHotplugDisk(oldDisk) {
 			continue
 		}
+		if oldDisk.Target.Bus == v1.DiskBusSATA {
```

Comment:
We only support hotplug for `scsi` and `virtio` so I think this condition would be better:

`if oldDisk.Target.Bus != v1.DiskBusVirtio && oldDisk.Target.Bus != v1.DiskBusSCSI {`

There are a ton of other busses that we don't support/test like ide, usb, etc